topics/proteomics/tutorials/metaproteomics/tutorial.md

```diff
@@ -560,29 +539,75 @@ As a final step we will use **Query Tabular** in a more sophisticated way to com
 >    - **Database Table**: Click on `+ Insert Database Table`
 >    - **Tabular Dataset for Table**: The `Gene Ontology Terms` file
 >
+>    Section **Filter Dataset Input**:
+>
+>    - **Filter Tabular Input Lines**: Click on `+ Insert Filter Tabular Input Lines`:
+>    - **Filter By**: Select `skip leading lines`
+>        - **Skip lines**: `1`
+>
 >    Section **Table Options**:
 >
 >    - **Specify Name for Table**: `go`
 >    - **Specify Column Names (comma-separated list)**: `aspect,go_id,description`
+>    - **Table Index**: Click on `+ Insert Table Index`:
+>        - **This is a unique index**: `No`
+>        - **Index on Columns**: `aspect,go_id`
 >
 >
 > 2. Repeat this step to have a second **Database Table**:
 >
 >    - **Database Table**: Click on `+ Insert Database Table`
->    - **Tabular Dataset for Table**: The **Unipept** normalized `tabluar`/`tsv` output
+>    - **Tabular Dataset for Table**: The **Unipept** `tabluar`/`tsv` output
+>
+>    Section **Filter Dataset Input**:
+>
+>    - **Filter Tabular Input Lines**: Click on `+ Insert Filter Tabular Input Lines`:
+>    - **Filter By**: Select `skip leading lines`
+>        - **Skip lines**: `1`
+>    - Add another Filter: Click on `+ Insert Filter Tabular Input Lines`:
+>    - **Filter By**: Select `prepend a line number column`
+>
+>    Section **Table Options**:
+>
+>    - **Specify Name for Table**: `bering_prot`
+>    - **Specify Column Names (comma-separated list)**: `id,peptide,uniprot_id,taxon_id,taxon_name,ec_references,go_references,refseq_ids,refseq_protein_ids,insdc_ids,insdc_protein_ids`
+>    - **Table Index**: Click on `+ Insert Table Index`:
+>        - **This is a unique index**: `No`
+>        - **Index on Columns**: `id,peptide`
+>
+> 3. Repeat this step to have another **Database Table**:
+>
+>    - **Database Table**: Click on `+ Insert Database Table`
+>    - **Tabular Dataset for Table**: The same **Unipept** `tabluar`/`tsv` output
 >
 >    Section **Filter Dataset Input**:
 >
 >    - **Filter Tabular Input Lines**: Click on `+ Insert Filter Tabular Input Lines`:
->    - **Filter By**: Select `comment char`
->        - **Ignore lines beginning with these characters**: Select `#`
+>    - **Filter By**: Select `skip leading lines`
+>        - **Skip lines**: leave blank
+>    - Add another Filter: Click on `+ Insert Filter Tabular Input Lines`:
+>    - **Filter By**: Select `prepend a line number column`
+>    - Add another Filter: Click on `+ Insert Filter Tabular Input Lines`:
+>    - **Filter By**: Select `select columns`
+>        - **enter column numbers to keep**: `1,7`
+>    - Add another Filter: Click on `+ Insert Filter Tabular Input Lines`:
+>    - **Filter By**: Select `normalize list columns, replicates row for each item in list`
+>        - **enter column numbers to normalize**: `2`
+>        - **List item delimiter in column**: leave blank
```

Comment:
blank does not work. should be comma?

Reply:
It should be a blank character (leaving it blank will automatically insert a blank character afaik, but I'll try to explain it more clearly).